src/Console_App_Calculator/Task1.cs

```diff
@@ -0,0 +1,94 @@
+﻿// <copyright file="Task1.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+namespace ConsoleAppCalculator
+{
+    /// <summary>
+    /// Class Task1 contains main method.
+    /// </summary>
+    public class Task1
+    {
+        /// <summary>
+        /// Options has all the arithmetic operations in order.
+        /// </summary>
+        private enum Options
+        {
+            Addition = 1,
+            Subtraction,
+            Multiplication,
+            Divison,
+        }
+
+        /// <summary>
+        /// Main method recieves two integers from user
+        /// </summary>
+        /// <param name="args">reads a line</param>
+        /// <exception cref="ArgumentNullException">Throws exception when args is null</exception>
+        public static void Main(string[] args)
+        {
+            if (args is null)
+            {
+                throw new ArgumentNullException(nameof(args));
+            }
+
+            int integer1, integer2, option, stop;
+            do
+            {
+                Console.WriteLine("Enter Integer1:");
```

Comment:
The user messages should be non technical for eg. you can Ask "Enter the first number"

Reply:
Changed as required